### PR TITLE
Small IT test fixes + version bump-ups

### PR DESCRIPTION
**Pull request by myronkscott**
tc-core version bump.

### CODE REVIEW

**Comment by mathieucarbou** [comment]
@myronkscott : can you please rebase ? ;-)

**Comment by myronkscott** [comment]
rebased

**Comment by saurabhagas** [comment]
Included the changes in #703 to avoid the rebase exercise. Closing this PR.

**Comment by mathieucarbou** [comment]
re-opening. Removed commits from #703. Build is not passing and also in this PR.

**Comment by saurabhagas** [comment]
Rebased this PR, and added angela version bump-up.

**Comment by mathieucarbou** [comment]
Rebased on master ;-)

File: pom.xml [hunk 1]
```diff
@@ -36,10 +36,10 @@
     <terracotta-apis.version>1.7.0-pre2</terracotta-apis.version>
     <terracotta-configuration.version>10.7.0-pre2</terracotta-configuration.version>
     <passthrough-testing.version>1.7.0-pre13</passthrough-testing.version>
-    <terracotta-core.version>5.7.0-pre28</terracotta-core.version>
+    <terracotta-core.version>5.7.0-pre29</terracotta-core.version>
     <tripwire.version>1.0.0-pre7</tripwire.version>
     <galvan.version>1.5.0-pre3</galvan.version>
-    <angela.version>3.0.23</angela.version>
+    <angela.version>3.0.24</angela.version>
```
Comment (mathieucarbou) [inline pom.xml:42]: @saurabhagas : this should be removed to another PR and update to the new version to release.
There are several issues in the PortManager use in Angela. See https://github.com/Terracotta-OSS/angela/pull/59

**Comment by saurabhagas** [comment]
Rebased on https://github.com/Terracotta-OSS/terracotta-platform/pull/707

**Comment by myronkscott** [comment]
obsolete.  #708
